<launch>
  <node pkg="nodelet" type="nodelet" name="image_and_pcl_manager"  args="manager" output="screen">
  	<param name="num_worker_threads" type="int" value="4" />
  </node>
    
  <node pkg="nodelet" type="nodelet" name="pcl_to_image" args="load image_cloud/Pcl_to_image_nodelet image_and_pcl_manager" output="screen">
  	<param name="sub_pcl" type="string" value="/velodyne/cloud" />
  	<param name="sub_img_info" type="string" value="/ladybug5/camera4/camera_info" />
  	<param name="image_frame_id"  type="string" value="ladybug_camera4" />
  	<param name="resize_x"  type="double" value="1" />
  	<param name="resize_y"  type="double" value="1" />
  </node>
  
</launch>
